### DISCUSSION TITLE
百度云上的crop511数据用pysot/datasets/dataset.py 中的_get_bbox函数转化坐标后部分存在groundtruth框不准问题？

### DISCUSSION THREAD

**Discussion by Joker1994k** [STVIR/pysot#155] [General] [answered]
您好，我想问下关于在百度云盘上的youtubeBB数据集问题。
根据pysot/datasets/dataset.py 中的_get_bbox函数，对train.json中的groundtruth坐标变换后与crop511中的图片有的并不能对应起来，请问是数据本身标注的原因还是crop的过程有其他设置？谢谢。
另外能否在百度云上提供下crop之前的图片数据集呢？想试试自己处理下，谢谢~~

**Comment by lb1100** [comment]
Can you provide some examples?
The original data is too large. You will have to download it yourself. Sorry that we can't help.

**Comment by Joker1994k** [comment]
> Can you provide some examples?
> The original data is too large. You will have to download it yourself. Sorry that we can't help.

There are some examples:
 ./crop511/train0036/19/MVF81zLXnp0
 ./crop511/train0050/2/VQxPemBgBvE
 ./crop511/val0034/1/KpHpGcL_jEc

I can't find which one is the tracking target based on the groundtruth。Thx

**Comment by lb1100** [comment, marked as answer]
I checked our data (VQxPemBgBvE). It seems our original data (frames from videos which have annotations) is wrong. It may be caused by downloading problems. Therefore our data may have some noises. However, I'm not sure how much noise there is. But I don't think this will have big influence. 
  We will look into this problem and refresh our data once it's done. But this may take some time since we also have the problem of downloading youtube videos.
